pkg/kvstore/etcd.go

```diff
@@ -340,6 +359,7 @@ func connectEtcdClient(config *client.Config, cfgPath string, errChan chan error
 		latestStatusSnapshot: "No connection to etcd",
 		stopStatusChecker:    make(chan struct{}),
 		extraOptions:         opts,
+		limiter:              rate.NewLimiter(rate.Limit(rateLimit), rateLimit), //TODO: figure out proper burst value
```

Comment:
Create a GH issue to track it and refer to it

Reply:
This comment is a leftover, basically the burst value doesn't really matter since we are doing bursts of 1 (we don't use `WaitN` at all). Setting it to `rateLimit` should work well.